assets/src/block-validation/components/error/get-error-source-title.js

```diff
@@ -25,16 +25,32 @@ export function getErrorSourceTitle( sources = [] ) {
 		}
 	}
 
+	return keyedSources;
+}
+
+/**
+ * Attempts to get the title of the plugin or theme responsible for an error.
+ *
+ * Adapted from AMP_Validated_URL_Post_Type::render_sources_column PHP method.
+ *
+ * @param {Object[]} sources Error source details from the PHP backtrace.
+ */
+export function getErrorSourceTitle( sources = [] ) {
+	const keyedSources = getKeyedSources( sources );
 	const output = [];
-	const uniquePluginNames = [ ...new Set( keyedSources.plugin.map( ( { name } ) => name ) ) ];
-	const muPluginNames = [ ...new Set( keyedSources[ 'mu-plugin' ].map( ( { name } ) => name ) ) ];
-	const combinedPluginNames = [ ...uniquePluginNames, ...muPluginNames ];
+	const uniquePluginNames = new Set( keyedSources.plugin.map( ( { name } ) => name ) );
+	const muPluginNames = new Set( keyedSources[ 'mu-plugin' ].map( ( { name } ) => name ) );
+	let combinedPluginNames = [ ...uniquePluginNames, ...muPluginNames ];
+
+	if ( combinedPluginNames.length > 1 ) {
+		combinedPluginNames = combinedPluginNames.filter( ( slug ) => ! slug.match( /^gutenberg/i ) );
```

Comment:
This will have the unexpected consequence of matching any plugin that starts with `gutenberg`, like Gutenberg Custom Fields: https://wordpress.org/plugins/gutenberg-custom-fields/

I don't think it's reasonable to expect normal users to have different version of Gutenberg installed with different slugs.

```suggestion
		combinedPluginNames = combinedPluginNames.filter( ( slug ) => slug !== 'gutenberg' );
```